src/abyssal_tome/app.py

```
@@ -666,26 +681,26 @@ async def route_change(route_event: ft.RouteChangeEvent) -> None:
             # Otherwise, append. This logic might need refinement for proper back navigation.
             new_view = ft.View(route_event.route, card_detail_view_content)
             if len(page.views) > 1 and page.views[-1].route != "/":
-                 page.views[-1] = new_view # Replace current detail view
+                 page.views[-1] = new_view  # Replace current detail view
```

Ruff (0.11.9)

Indentation is not a multiple of 4

(E111)